src/main/interface/settings.c

```diff
@@ -637,6 +637,7 @@ const clivalue_t valueTable[] = {
 
 // PG_PID_PROFILE
     { "dterm_lowpass_type",         VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_LOWPASS_TYPE }, PG_PID_PROFILE, offsetof(pidProfile_t, dterm_filter_type) },
+	{ "dterm_lowpass_2",              VAR_INT16  | PROFILE_VALUE, .config.minmax = { 0, 16000 }, PG_PID_PROFILE, offsetof(pidProfile_t, dterm_lpf_2_hz) },
```

Comment:
Should be `dterm_lowpass2_type`

Reply:
With this implementation, the type can't be changed, it's set to a PT1, so the user enters the value in Hz.  

Maybe it should be 'dterm_lowpass_2_hz'

Reply:
No underscore before 2, please, to conform with gyro lowpass and notch

Reply:
done